.circleci/config.yml

```diff
@@ -31,6 +31,16 @@ jobs:
       PYTEST_ARGS: -k DESKTOP -m run_targeting -n 2
     steps:
       - checkout
+      - run:
+          name: Check file paths
+          command: |
+            if git diff --name-only main HEAD | grep -E 'app/experimenter/targeting|app/experimenter/experiments'
```

Comment:
It's too bad we can't narrow it down even further.... I guess we could move the targeting logic out of the experiment model and into a mixin in the targeting app....  Maybe we can do that as a followup.

Reply:
We can narrow it down to the file name itself can't we?

Reply:
No it's like specific methods inside the file that we care about in this context, but I was also already thinking of moving that bit of logic into the targeting app anyways.